ixwebsocket/IXUdpSocket.cpp

```diff
@@ -93,4 +113,11 @@ namespace ix
         return (ssize_t)::sendto(
             _sockfd, buffer.data(), buffer.size(), 0, (struct sockaddr*) &_server, sizeof(_server));
     }
+
+    ssize_t UdpSocket::recvfrom(void* buffer, size_t length)
+    {
```

Comment:
Can you put the sizeof(_server) directly in the function call ?
I am afraid of compiler warnings.